lib/app/views/test_case.erb

```diff
@@ -70,10 +70,15 @@
                   <% if sequence_results[test_case.id].nil? %>
                     <%= test_case.sequence.test_count %> tests
                   <% else %>
-                    <%= sequence_results[test_case.id].required_passed %>/<%= sequence_results[test_case.id].required_total%> Required Tests Passed
+                    <%= sequence_results[test_case.id].required_passed %>/<%= sequence_results[test_case.id].required_total  - sequence_results[test_case.id].required_omitted %> Required Tests Passed
+
                     <% if sequence_results[test_case.id].optional_total > 0%> -
```

Comment:
```
<% if sequence_results[test_case.id].optional_total > 0%> -
  <%= sequence_results[test_case.id].optional_passed%>/<%= sequence_results[test_case.id].optional_total%> Optional Tests Passed
<% end%>
```

We repeat this a few times to handle required, optional and omitted tests.  Might be worth looping over it once for each of the different cases instead.